src/core/db/manager.py

```diff
@@ -526,6 +931,14 @@ def create_user(username, password, create_db=True):
         return res
 
     @staticmethod
+    def create_user_database(username):
+        """Creates the database and data folder for a user."""
+        with _superuser_connection() as conn:
+            res = conn.create_user_database(username=username)
+            DataHubManager.create_user_data_folder(username)
```

Comment:
What's the point of creating the user data folder here? If I called `create_user_database`, I wouldn't initially expect it to `create_user_data_folder`; especially since there's another public method that does that.